roomserver/storage/postgres/rooms_table.go

```diff
@@ -42,13 +43,13 @@ CREATE TABLE IF NOT EXISTS roomserver_rooms (
     state_snapshot_nid BIGINT NOT NULL DEFAULT 0,
     -- The version of the room, which will assist in determining the state resolution
     -- algorithm, event ID format, etc.
-    room_version BIGINT NOT NULL DEFAULT 1
+    room_version TEXT NOT NULL DEFAULT '1'
```

Comment:
Please don't default here, as it means to update the default room version we need to remember to do it here and `GetDefaultRoomVersion()`.

Reply:
The main reason for this was because a `m.room.create` event that doesn't contain a `room_version` option is typically a v1 room. Any room of a later version must contain `room_version` therefore we will always have something to overwrite the default with in those cases.

Reply:
I'd rather that check resided in code rather than having important semantics tucked away in an easily missed create table statement. Especially since this line gives the impression that by default we support v1 (e.g room version is an optional param) whereas what it's really saying is "by default, if and only if the room version is missing, set it to v1".

Reply:
I have removed the default for now. We can handle it in code instead.